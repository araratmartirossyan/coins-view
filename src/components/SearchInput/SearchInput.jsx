import React from "react";

import TextField from "@material-ui/core/TextField";

export default function SearchInput({ className, search, handleChangeSearch }) {
  return (
    <div className={className}>
      <TextField
        id="outlined-basic"
        size="small"
        label="Search"
        defaultValue={search}
        onChange={handleChangeSearch}
        variant="outlined"
      />
    </div>
  );
}
