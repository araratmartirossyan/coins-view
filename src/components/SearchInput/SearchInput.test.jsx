import React from 'react'
import renderer from 'react-test-renderer'

import SearchInput from './SearchInput'

it('renders correctly Search Input', () => {
  const input = renderer.create(<SearchInput />).toJSON()
  expect(input).toMatchSnapshot()
})
