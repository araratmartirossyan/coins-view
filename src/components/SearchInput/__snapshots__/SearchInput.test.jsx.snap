// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Search Input 1`] = `
<div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense MuiInputLabel-outlined"
      data-shrink={false}
      htmlFor="outlined-basic"
      id="outlined-basic-label"
    >
      Search
    </label>
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
        disabled={false}
        id="outlined-basic"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
      />
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-59 MuiOutlinedInput-notchedOutline"
      >
        <legend
          className="PrivateNotchedOutline-legendLabelled-61"
        >
          <span>
            Search
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</div>
`;

exports[`renders correctly when there are no items 1`] = `
<div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense MuiInputLabel-outlined"
      data-shrink={false}
      htmlFor="outlined-basic"
      id="outlined-basic-label"
    >
      Search
    </label>
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
        disabled={false}
        id="outlined-basic"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
      />
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-59 MuiOutlinedInput-notchedOutline"
      >
        <legend
          className="PrivateNotchedOutline-legendLabelled-61"
        >
          <span>
            Search
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</div>
`;
